src/main/java/net/sf/jabref/external/AttachFileAction.java

```diff
@@ -25,7 +26,8 @@ public AttachFileAction(BasePanel panel) {
     @Override
     public void action() {
         if (panel.getSelectedEntries().size() != 1) {
-            return; // TODO: display error message?
+            panel.output(Localization.lang("No entries or multiple entries selected."));
```

Comment:
I would prefer if you could clean up all the related messages as well in this PR. Right now the output is just a statement and not an error message.